test/tests/rackhd11/test_rackhd11_api_config.py

```diff
@@ -44,18 +42,25 @@ def test_api_11_config_httpendpoints(self):
 
     def test_api_11_config_patch(self):
         api_data_save = fit_common.rackhdapi('/api/1.1/config')['json']
-        data_payload = {"CIDRNet": "127.0.0.1/22"}
+        if ("logColorEnable" not in api_data_save):
+            api_data_save['logColorEnable'] = False
+        if (api_data_save['logColorEnable'] is True):
+            data_payload = {"logColorEnable": False}
+        else:
+            data_payload = {"logColorEnable": True}
         api_data = fit_common.rackhdapi("/api/1.1/config", action="patch", payload=data_payload)
         self.assertEqual(api_data['status'], 200, "Was expecting code 200. Got " + str(api_data['status']))
         for item in api_data['json']:
-            if fit_common.VERBOSITY >= 2:
-                print "Checking:", item
             self.assertNotEqual(item, '', 'Empty JSON Field:' + item)
         self.assertEqual(api_data['status'], 200, "Was expecting code 200. Got " + str(api_data['status']))
+        if ("logColorEnable" in api_data_save and api_data_save['logColorEnable'] is True):
+            self.assertEqual(api_data['json']['logColorEnable'], False, "Incorrect patched value for 'logColorEnable'")
+        else:
+            self.assertEqual(api_data['json']['logColorEnable'], True, "Incorrect patched value for 'logColorEnable'")
```

Comment:
The same error message is printed in both assert cases, this will be hard to debug later.  Can you differentiate the failures?